Dockerfile

```diff
@@ -0,0 +1,43 @@
+# extend node alpine base image
+# using 8-alpine allows renovate to update
+FROM node:8-alpine
+
+# TODO: use custom renovation?
+ENV RENOVATE_VERSION 10.6.1
```

Comment:
I woudl expect a new image version to get pushed for every release, so the version and Dockerfile version would be tighlty coupled. Seems a bit weird to have that configurable?

Reply:
This is an old approach I will probably abandon - we can discuss instead in the issue you raised today